### PR TITLE
cleaned up folder structure and updated readmes

### PR DESCRIPTION
**Pull request by wlfghdr**

### CODE REVIEW

**Comment by agrimmer** [comment]
The changes would break links in our docu https://docs.dynatrace.com/docs/manage/ownership/ownership-app#import-teams

**Comment by wlfghdr** [comment]
ok so we need to change docu links - i was already afraid of that.
but the structure needed to be updated.

**Comment by wlfghdr** [comment]
in the meanwhile i re-added the (duplicate) template folder until we refactored the deep links.

File: samples/Team-Import/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Ownership Team Imports Templates
+
+Within this directory, you'll find templates designed for importing Ownership teams through Workflows. Our data sources for this process include Microsoft Entra ID, ServiceNow and a custom file hosted on GitHub.
```
Comment (stefaneberl) [inline samples/Team-Import/README.md:3]:
```suggestion
Within this directory, you'll find templates designed for importing Ownership teams through workflows. Our data sources for this process include Microsoft Entra ID, ServiceNow and a custom file hosted on GitHub.
```

File: samples/Team-Import/import_teams_from_custom_source.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        states:
+          fetch_teams_from_external_source: OK
+    fetch_teams_from_external_source:
+      name: fetch_teams_from_external_source
+      description: Build a custom task running js Code
+      action: dynatrace.automations:run-javascript
+      input:
+        script: >-
+          const apiUrl =
+          'https://raw.githubusercontent.com/Dynatrace/Dynatrace-workflow-samples/main/samples/Ownership/Team-Import/example-teams.json';
```
Comment (agrimmer) [inline samples/Team-Import/import_teams_from_custom_source.yaml:37]: Link needs to be fixed: Remove folder "Ownership/"

**Comment by agrimmer** [comment]
I've prepared a fix for the Dynatrace documentation.
I suggest that you again remove the duplicate content.

File: samples/Team-Import/import_teams_from_custom_source.yaml (hunk 2 above)
Comment (wlfghdr) [reply samples/Team-Import/import_teams_from_custom_source.yaml:37]: removed.

**Comment by wlfghdr** [comment]
I will change the URL behind as soon as I merge here ... (for https://dt-url.net/nj03quj )

**Review by stefaneberl — APPROVED** [review]

**Review by agrimmer — APPROVED** [review]
LGTM

**Pull request merged**